extensions/wikidata/src/org/openrefine/wikidata/editing/EditBatchProcessor.java

```diff
@@ -67,6 +70,14 @@
     private int globalCursor;
     private Map<String, EntityDocument> currentDocs;
     private int batchSize;
+    private static final String MAX_LAG_KEY = "maxLag";
```

Comment:
I would replace `"maxLag"` by something a bit more specific, to make sure there is no collision with anything else: for instance `"wikibase:upload:maxLag"`.

Reply:
will improve the naming.